### PR TITLE
4 get api review review

### PR DESCRIPTION
**Pull request by JulianaMatovu**
Handled happy path, 404 and 400 errors for review_id param

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -40,8 +40,54 @@ describe("GET /api/categories", () => {
...
+        .get("/api/reviews/33")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("review_id not found");
+        });
+    });
+    test("returns status: 400 if review_id is invalid", () => {
+      return request(app)
+        .get("/api/reviews/banana")
+        .expect(400);
```
Comment (two-triangles) [inline __tests__/app.test.js]: We need to make assertion for the error message that you have created for this 400 error

Comment (JulianaMatovu) [reply __tests__/app.test.js]: https://github.com/JulIanaMatovu/Backend-Project-NC-Games/pull/3/commits/3311f4aa47b78d42a41844d77a61224a50b7ab63

Hi Saleh, I have added this now

File: __tests__/app.test.js [hunk 2]
```diff
@@ -40,8 +40,57 @@ describe("GET /api/categories", () => {
...
+          });
+          expect(body).toHaveProperty("review_id");
+          expect(body).toHaveProperty("title");
+          expect(body).toHaveProperty("review_body");
+          expect(body).toHaveProperty("designer");
+          expect(body).toHaveProperty("review_img_url");
+          expect(body).toHaveProperty("votes");
+          expect(body).toHaveProperty("category");
+          expect(body).toHaveProperty("owner");
+          expect(body).toHaveProperty("created_at");
```
Comment (emilyb93) [inline __tests__/app.test.js]: This is unncessary code as you are testing for all of these things in a more specific way above by checking it equals the exact values of the above object

**Review by emilyb93 — APPROVED** [review]
Maybe just remove that doubled up test and this is good to go

**Pull request merged**